review/tests/test_review.py

```diff
@@ -0,0 +1,49 @@
+import pytest
+from datetime import timedelta
+from django.utils import timezone
+from review.models import Review
+
+
+@pytest.fixture
+def review():
+    # Checks if the review was created already
+    review = Review.objects.filter(client_id='client_id', professional_id='professional_id')
+    # Checks if there are at least one object of the same Review and then deletes it
+    if len(review) > 0:
+        review.delete()
+    # Creates and returns a new review object
+    now = timezone.now()
+    review = Review.objects.create(rating='4', description='Creating a test review...',
+                                   date_posted=now - timedelta(days=50),
+                                   client_id='client_id', professional_id='professional_id')
+    yield review        # Returns review object
+    review.delete()     # TearDown
```

Comment:
This is volatile data, you can avoid the delete

Reply:
> This is volatile data, you can avoid the delete

Fixed